code/src/css/button.css

```diff
@@ -0,0 +1,17 @@
+button {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 10px;
+  border-radius: 40px;
+  background-color: lightgreen;
+  font-weight: 700;
+  font-family: 'Gochi Hand', cursive;
+  font-size: 24px;
+  color: purple;
```

Comment:
Super super small detail, but I saw that you mentioned font-family further down as well. Maybe put them in body so you don't need to repeat? (but if you only mention it twice maybe there's no need for that?)